tests/integration/defs/accuracy/test_llm_api_pytorch_multimodal.py

```
@@ -589,6 +592,71 @@ def test_nvfp4(self, ep_size, attention_dp):
+    # answer; extract_kimi_k3_mmmu_answer reads the channel and falls back to
+    # the K2.5 cascade for channel-less outputs. No chat_template_kwargs: K3
+    # thinks by default. preserve_caller_max_tokens keeps max_tokens=16384
+    # over lm-eval's 512-token MMMU default — without it the CoT is truncated
+    # before the response channel opens, the extractor silently falls back,
+    # and the score sinks toward the K2.5 reference floor (see
+    # references/mmmu.yaml): treat such a score with clean logs as a harness
+    # failure, not a model regression.
+    EXTRA_EVALUATOR_KWARGS = dict(
+        post_process_fn=extract_kimi_k3_mmmu_answer,
+        preserve_caller_max_tokens=True,
+    )
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)